onebusaway-gtfs-transformer/src/main/java/org/onebusaway/gtfs_transformer/impl/WSFBlockResolutionStrategy.java

```diff
@@ -151,8 +151,8 @@ private void setBlockIdsFromSchedResponse(SchedResponse resp) {
       String arrive = stc.getArrivingTerminalID().toString();
 
       for (SchedTime sched : schedTime(stc)) {
-        long time = ts(sched.getDepartingTime());
```

Comment:
@sdjacobs I think the optimization is here - if you look at `private long ts(XMLGregorianCalendar xgc) {...}` method, within that method it instantiates a `GregorianCalendar` to get the time, but then throws it away and just returns a `long`.  Then, in `resolve()` another `Calendar` object is instantiated again.  This cuts out the instatiation in `resolve()` by passing in an existing instance of `Calendar` into `resolve()`.

@skjolber Did I get that right?

Reply:
@barbeau  Yes.